src/v3/components/user_account_item/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, IconButton, Typography } from '@mui/material';
```

Comment:
Sorry @FussuChalice for coming to this again 😁, we have our custom Typography component that should be used instead of the default one from MUI. Could you please adjust this import?